frontend/src/app/specification-version/graphiql-wrapper.component.ts

```diff
@@ -40,21 +40,24 @@ export class GraphiQLWrapperComponent extends VersionViewComponent implements On
     return !!this.graphiql && !!this.specification;
   }
 
-  public graphQLFetcher(graphQLParams) {
-    return fetch('http://localhost:4000/graphql', {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(graphQLParams)
-    }).then(response => response.json())
-      .catch(() => 'An error occurred with the network request. ' +
-        '(Check the GraphQL service is running, and accepts POST requests)');
+  public graphQLFetcher(apiEndpoint?: string) {
+    return (graphQLParams) => {
+      return fetch(apiEndpoint ? apiEndpoint : 'http://localhost:4000/graphql', { // As a fallback, use express-graphql's default
```

Comment:
How about we use a new endpoint in the backend as a fallback that will only return an error?

Reply:
I much prefer this idea 😎